lib/inferno/apps/web/controllers/test_sessions/create.rb

```diff
@@ -10,7 +10,9 @@ class Create < Controller
 
           def handle(req, res)
             params = req.params.to_h
-            params.merge!(JSON.parse(req.body.string).symbolize_keys) unless req.body.string.blank?
+            if !req.body.string.blank? && !req.env['CONTENT_TYPE'].include?('multipart/form-data')
```

Comment:
Use `present?` instead of `!blank?`.

Why check that the content type isn't form data instead of checking whether the content type is json? Also, that isn't the only content type used by forms.

Reply:
Sounds good, I've changed it to just check for json instead before attempting to parse as a json.  I'm still learning in's and out's of request handling -- is there any form data that is non json that could need additional handling?  My understanding is that Hanabi will handle most forms automatically.